src/web/app/pages-instructor/instructor-course-details-page/instructor-course-details-page.component.ts

```diff
@@ -7,9 +7,19 @@ import { CourseService } from '../../../services/course.service';
 import { HttpRequestService } from '../../../services/http-request.service';
 import { NavigationService } from '../../../services/navigation.service';
 import { StatusMessageService } from '../../../services/status-message.service';
-import { MessageOutput } from '../../../types/api-output';
+import { StudentService } from '../../../services/student.service';
+import {
+  Course,
+  Instructor,
+  InstructorPrivilege,
+  Instructors,
+  MessageOutput,
+  Student,
+  Students,
+} from '../../../types/api-output';
 import { ErrorMessageOutput } from '../../error-message-output';
-import { StudentListSectionData } from '../student-list/student-list-section-data';
+import { Intent } from '../../Intent';
+import { StudentListSectionData, StudentListStudentData } from '../student-list/student-list-section-data';
 
 interface CourseAttributes {
```

Comment:
I think you don't need the interface anymore?

Reply:
Alright. I'll use the api-output types directly